modules/tinymce/src/themes/silver/main/ts/ui/toolbar/button/ToolbarButtons.ts

```diff
@@ -275,7 +294,21 @@ const renderSplitButton = (spec: Toolbar.ToolbarSplitButton, sharedBackstage: Ui
         comp.getSystem().getByDom(button).each((buttonComp) => Toggling.set(buttonComp, state));
       });
     },
-    isActive: () => SelectorFind.descendant(comp.element, 'span').exists((button) => comp.getSystem().getByDom(button).exists(Toggling.isOn))
+    isActive: () => SelectorFind.descendant(comp.element, 'span').exists((button) => comp.getSystem().getByDom(button).exists(Toggling.isOn)),
+    setText: (text: string) =>
+      SelectorFind.descendant(comp.element, 'span').each((button) =>
```

Comment:
I wonder if this could just be emitted on the `comp` instead of the `buttonComp` the event broadcast should be sent down to all the descendants. Or do you think there is a risk that something on the actual comp would also listen to the same broadcasts?

Reply:
Unfortunately that doesn't work, I know it because a test case fails, but I don't know exactly why it doesn't work.